packages/diagrams/backend/sirius-components-diagrams-layout-api/src/main/java/org/eclipse/sirius/components/diagrams/layout/api/Anchor.java

```diff
@@ -0,0 +1,51 @@
+/*******************************************************************************
+ * Copyright (c) 2023 Obeo.
+ * This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v2.0
+ * which accompanies this distribution, and is available at
+ * https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     Obeo - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.sirius.components.diagrams.layout.api;
+
+import org.eclipse.sirius.components.diagrams.layoutdata.Position;
+import org.eclipse.sirius.components.diagrams.layoutdata.Size;
+
+/**
+ * Possible values for anchoring a box inside a container.
+ *
+ * @author pcdavid
+ */
+public enum Anchor {
```

Comment:
May not be used anymore, but was used to stick a box(node, label) on a side of a container

Reply:
For reference, this was used here: https://github.com/eclipse-sirius/sirius-components/pull/1841/commits/6d8a74ea5bcdbdd8cc213fbb0c9e43d107a7f313#diff-e1426eec0f29c695bdec886f0597e622696d9de0146b217b3de8f58f1db2a03cR55